examples/spa/yarn.lock

```diff
@@ -1161,9 +1161,8 @@
     strip-json-comments "^3.1.1"
 
 "@fingerprintjs/fingerprintjs-pro-react@link:../..":
-  version "1.3.1"
-  dependencies:
-    "@fingerprintjs/fingerprintjs-pro-spa" "^0.4.0"
+  version "0.0.0"
```

Comment:
Do you have any idea why this happens? And what is the sudden dependency `scheduler`?

Reply:
I'm not sure, I can imagine that it happens because the dependency is linked from the root directory. `scheduler` probably comes from `react`.